core/ui/compose/demo/src/main/java/app/k9mail/core/ui/compose/demo/MainView.kt

```diff
@@ -0,0 +1,61 @@
+package app.k9mail.core.ui.compose.demo
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import app.k9mail.core.ui.compose.theme.K9Theme
+import app.k9mail.core.ui.compose.theme.MainTheme
+import app.k9mail.core.ui.compose.theme.ThunderbirdTheme
+
+@Composable
+fun MainView() {
+    Column(
+        modifier = Modifier.fillMaxSize()
+    ) {
+        K9Theme {
+            Surface(modifier = Modifier.fillMaxWidth(), color = MainTheme.colors.background) {
+                Column {
+                    Text(text = "Hello K-9!")
+                    Image(painter = painterResource(id = MainTheme.images.logo), contentDescription = "log")
+                }
+            }
```

Comment:
Extracting this to a separate composable function would make it more obvious that all of these blocks are the same (except for the message).

Note: The content description contains a typo ("log" vs. "logo").